src/configuration.ts

```diff
@@ -142,6 +141,35 @@ export function findConfigurationPath(suppliedConfigFilePath: string | null, inp
 }
 
 /**
+ * Find a file by name in a directory or any ancestory directory.
+ * This is case-insensitive, so it can find 'TsLiNt.JsOn' when searching for 'tslint.json'.
+ */
+function findup(filename: string, directory: string): string | undefined {
```

Comment:
In fact we are calling it a lot. When not passing `-c` on the command line, `Runner` calls `findConfiguration` for every time is sees a file in another directory than the previous.